### PR TITLE
chore(deps): update to FLUID-6145 branch of Infusion

### PR DESCRIPTION
**Pull request by greatislander**
Updates UIO integration provided by this plugin to use the FLUID-6145 line of Infusion.

Steps to test:

1. Check out this branch.
2. `npm install && npm start`.
3. Open http://localhost:8080.
4. Verify that UI Options adjusters work as expected.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     },
     "dependencies": {
         "@11ty/eleventy": "0.11.1",
-        "infusion": "3.0.0-dev.20200326T173810Z.24ddb2718",
+        "infusion": "3.0.0-dev.20201007T193943Z.440be20ec.FLUID-6145",
```
Comment (amb26) [inline package.json:44]: Most recent stable release is 3.0.0-dev.20201113T153152Z.32176dcbe.FLUID-6145

Comment (greatislander) [reply package.json]: Thanks!

File: .eleventy.js [hunk 2]
```diff
@@ -1,61 +1,16 @@
...
-Copyright the eleventy-plugin-fluid copyright holders.
-
-See the AUTHORS.md file at the top-level directory of this distribution and at
-https://github.com/fluid-project/eleventy-plugin-fluid/raw/main/AUTHORS.md.
-
-Licensed under the New BSD license. You may not use this file except in compliance with this License.
-
-You may obtain a copy of the New BSD License at
-https://github.com/fluid-project/eleventy-plugin-fluid/raw/main/LICENSE.md.
-*/
```
Comment (jobara) [inline .eleventy.js:11]: Was the removal of the copyright statement intentional?

Comment (greatislander) [reply .eleventy.js:11]: This is actually a new file. The Eleventy plugin's main file has been renamed to `index.js` and I added this file solely for the purposes of testing the UIO integration within this repository. As a test file which won't be used in projects which include this as a dependency, I didn't think to add a copyright statement but if you think it's necessary I can.

File: src/config/uio-assets.json [hunk 3]
```diff
@@ -14,6 +14,7 @@
     "src/framework/enhancement/css/",
     "src/framework/core/css/",
     "src/framework/core/images/",
+    "src/lib/hypher/patterns",
```
Comment (jobara) [inline src/config/uio-assets.json:17]: It doesn't look like syllabification is being used, so this isn't necessary. Although leaving it in would make it easier to configure UIO to use syllabication in the future.

File: .eleventy.js (hunk 2 above)
Comment (jobara) [reply .eleventy.js:11]: Thanks for the clarification. We tend to add copyright statements to all of the JS files, so might as well.

File: src/config/uio-assets.json (hunk 3 above)
Comment (greatislander) [reply src/config/uio-assets.json:17]: @jobara That's the idea.

**Pull request merged**